numba/cuda/cudadrv/libs.py

```diff
@@ -1,23 +1,28 @@
 """Cudatoolkit libraries lookup utilities.
 
-Cudatoolkit libraries can be available via the `cudatoolkit` conda package or a
-user supplied location from CUDA_HOME or a system wide location.
+Cudatoolkit libraries can be available via either:
+
+- the `cudatoolkit` conda package,
+- a user supplied location from CUDA_HOME,
+- a system wide location,
+- package-specific locations (e.g. the Debian NVIDIA packages),
+- or can be discovered by the system loader.
 """
+
 from __future__ import print_function
 import os
 import sys
 import ctypes
-import platform
 
 from numba.findlib import find_lib
 from numba.cuda.cuda_paths import get_cuda_paths
 
 if sys.platform == 'win32':
-    _dllopener = ctypes.WinDLL
+    _dllnamepattern = 'lib%s.dll'
```

Comment:
On windows, the dll name do not have the 'lib' prefix. The cudatoolkit eula has a table of all the shared lib names: https://docs.nvidia.com/cuda/eula/index.html#attachment-a